src/python/pants/engine/isolated_process.py

```diff
@@ -10,24 +10,29 @@
 
 from pants.engine.fs import Digest
 from pants.engine.rules import RootRule, rule
-from pants.util.objects import Exactly, TypedCollection, datatype
+from pants.util.objects import Exactly, SubclassesOf, TypedCollection, datatype
 
 
 logger = logging.getLogger(__name__)
 
 _default_timeout_seconds = 15 * 60
 
 
+_string_type = SubclassesOf(text_type)
```

Comment:
Is it worth defining these in `pants.util.objects` as stopgaps until `Python3 Typing TM`?

Reply:
Yes!